app/assets/javascripts/editorToolbar.js

```diff
@@ -36,13 +16,8 @@ const progressAll = (elem, data) => {
 // attach eventListeners on document.load for toolbar rich-text buttons & image upload .dropzones
 $(function() {
   // for rich-text buttons (bold, italic, header, and link):
-  // click eventHandler that assigns $D.selected to the appropriate comment form
-  // on pages with multiple comments, $D.selected needs to be accurate so that rich-text changes (bold, italic, etc.) go into the right comment form
   $('.rich-text-button').on('click', function(e) {
-    const { textArea, preview, dSelected } = getEditorParams(e.target);
```

Comment:
completely removed this `getEditorParams` helper function and everywhere it's called!